imports/plugins/core/product/server/no-meteor/schemas/product.graphql

```diff
@@ -0,0 +1,255 @@
+"A Reaction product"
+type Product {
+  "Product ID"
+  _id: ID!
+
+   "The date and time at which this product was created"
+  createdAt: DateTime!
+
+  "Hash to compare with publishedProductHash, for use to see it published and unpublished products are the same"
+  currentProductHash: String
+
+  "The full product description, which may have newline characters in it"
+  description: String
+
+  "True if this product has been deleted. Typically, deleted products are not returned in queries."
+  isDeleted: Boolean!
+
+  "True if this product should be shown to shoppers. Typically, non-visible products are not returned in queries."
+  isVisible: Boolean!
+
+  "The product description to use for page `description` meta element in HTML"
+  metaDescription: String
+
+  "Arbitrary additional metadata about this product"
+  metafields: [Metafield]
+
+  "The country of origin"
+  originCountry: String
+
+  "Subtitle"
+  pageTitle: String
+
+  "An arbitrary product type value, such as from an external system"
+  productType: String
+
+  "The date and time at which this product was published"
+  publishedAt: DateTime
+
+  "Hash to compare with currentProductHash, for use to see it published and unpublished products are the same"
+  publishedProductHash: String
+
+  "The shop to which this product belongs"
+  shop: Shop!
+
+  "A URL-safe and human-readable string that uniquely identifies this product"
+  slug: String
+
+  "Holds metadata specific to a specific social network service"
+  socialMetadata: [SocialMetadata]
+
+  "When a shopper purchases this product, what types of fulfillment can they choose from?"
+  supportedFulfillmentTypes: [FulfillmentType]!
+
+  "The list of tag IDs that have been applied to this product"
+  tagIds: [ID]
+
+  "The list of tags that have been applied to this product"
+  tags(
+    "Return only results that come after this cursor. Use this with `first` to specify the number of results to return."
+    after: ConnectionCursor,
+
+    "Return only results that come before this cursor. Use this with `last` to specify the number of results to return."
+    before: ConnectionCursor,
+
+    "Return at most this many results. This parameter may be used with either `after` or `offset` parameters."
+    first: ConnectionLimitInt,
+
+    "Return at most this many results. This parameter may be used with the `before` parameter."
+    last: ConnectionLimitInt,
+
```

Comment:
Should add the `offset` field here to the schema like here: https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/catalog/server/no-meteor/schemas/schema.graphql#L421

Reply:
Done.